### ISSUE TITLE
Vision document does not say how vision is achieved

### ISSUE THREAD

**Issue by msporny** [w3c/AB-public#212]
Based on a recent review of https://www.w3.org/TR/2024/NOTE-w3c-vision-20241018/ , the Vision document doesn't mention how the vision is achieved. That is, it speaks to "Operational Principles for W3C" yet never links to our process, patent policy, etc... the things that make those Operating Principles achievable. We don't need to go into any amount of depth, but we should make some references to W3C Process and other operational documentation.

The mechanisms that we have in place for achieving the vision is important to refer to otherwise the reader might be left with the impression that there is no mechanism in place that guides the implementation of the vision.

**Comment by michaelchampion** [comment]
> the Vision document doesn't mention how the vision is achieved. 

I have the same reaction, and over the years have tried to make various suggestions on what to say about how W3C could nudge the real world Web to be less toxic. BUT  it's clear there's not really much consensus within W3C on what could be done, and even whether things like "business models that mine and sell detailed user data, without people’s awareness or consent" are just the price we must pay for the benefits of the web we have today.

So at this point, I'd prefer to see the AC declare victory on the Vision more or less as it stands, and focus on its other, more urgent, priorities.

**Comment by cwilso** [comment]
I actually think we DO need to go into depth here, but maybe not in this document; and it is really, really important to get basic principles established before making them operational.  (We've been down this rabbithole repeatedly in the past couple of years.  If you jump straight to operationalizing, you find that people don't have a shared understanding of the core principle.)

Which is to say, I agree with your point, but would prefer to establish the guiding light before clarifying exactly how we point the ship toward it.

**Comment by msporny** [comment]
I don't think we need to elaborate on any of the details in the W3C Process Document or Patent Policy. All I'm asking for is a informative ref to `[W3C-PROCESS]` in the doc somewhere. No need to elaborate, just help people follow their nose to the W3C Process document (and perhaps some of the other legal / policy documents). For example:

> [5. Operational Principles for W3C](https://www.w3.org/TR/2024/NOTE-w3c-vision-20241018/#op-principles)
> In order to fulfill our Vision, we will follow these operational principles:
> ...
> *For a more details on these principles, see [W3C-PROCESS] and [W3C-PATENT-POLICY].`

**Comment by chrisn** [comment]
I debated whether to raise a separate issue for this or just comment here. My reading of the Introduction section ("We must do better. We must take steps to address these consequences in the standards we create") leads me to expect a manifesto for how to address the harms described. But it doesn't then follow through on that. The referenced EWP concepts are fine (but why highlight a subset?). I understand the rationale for linking rather than duplicating the principles, but it means the Vision itself doesn't contain much detail, and it then goes on to describe operational principles that also don't speak to addressing the harms.
